src/atoms/Checkbox/index.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+const Checkbox = styled.input.attrs({
+  type: 'checkbox'
+})`
+  margin-right: 10px;
+`
+
+Checkbox.displayName = 'Checkbox'
+
+Checkbox.propTypes = {
+  name: PropTypes.string,
```

Comment:
You still need proptypes if not used with `styled-components`?

Reply:
I think, we can delete it.